### PR TITLE
add controls to experiment cards

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by johngruen** [comment]
Fixes #1525

File: frontend/src/app/components/ExperimentRowCard.js [hunk 1]
```diff
@@ -69,6 +70,17 @@ export default class ExperimentRowCard extends React.Component {
     );
   }
 
+  renderManageButton(enabled, hasAddon) {
+    if (enabled && hasAddon) {
+      return (
+        <div className="button card-control secondary" data-l10n-id="experimentCardManage">Manage</div>
```
Comment (chuckharmston) [inline frontend/src/app/components/ExperimentRowCard.js]: `Manage` is strange wording. What sort of management are users doing here?

Comment (chuckharmston) [inline frontend/src/app/components/ExperimentRowCard.js:76]: `Manage` is strange wording. What sort of management are users doing here?

**Review by chuckharmston — APPROVED** [review]
Looks good other than a nit about the wording. I'll approve it and leave it to you to self-merge.

**Pull request merged**